WindowsServerDocs/administration/server-manager/server-manager.md

```diff
@@ -50,11 +50,11 @@ To manage servers that are running Windows Server releases older than Windows Se
 #### Manage remote computers from a client computer
 The Server Manager console is included with [Remote Server Administration Tools](https://go.microsoft.com/fwlink/?LinkID=404281) for Windows 10. Note that when Remote Server Administration Tools is installed on a client computer, you cannot manage the local computer by using Server Manager; Server Manager cannot be used to manage computers or devices that are running a Windows client operating system. You can only use Server Manager to manage Windows-based servers.
 
-|Server Manager Source Operating System|Targeted at Windows Server 2016|Targeted at  Windows Server 2012 R2 |Targeted at  Windows Server 2012 |Targeted at  Windows Server 2008 R2  or  Windows Server 2008 |Targeted at Windows Server 2003|
-|-------------------------------|--------------------------------------------|---------------------------------------|------------------------------------|-----------------------------------------------------------------------|------------------|
-|Windows 10 or Windows Server 2016|Full support|Full support|Full support|After [Software and configuration requirements](#software-and-configuration-requirements) are satisfied, can perform most management tasks, but no role or feature installation or uninstallation|Not supported|
-|Windows 8.1 or  Windows Server 2012 R2 |Not supported|Full support|Full support|After [Software and configuration requirements](#software-and-configuration-requirements) are satisfied, can perform most management tasks, but no role or feature installation or uninstallation|Limited support; online and offline status only|
-|Windows 8 or  Windows Server 2012 |Not supported|Not supported|Full support|After [Software and configuration requirements](#software-and-configuration-requirements) are satisfied, can perform most management tasks, but no role or feature installation or uninstallation|Limited support; online and offline status only|
+|Server Manager Source Operating System|Targeted at Windows Server 2016|Targeted at  Windows Server 2012 R2 |Targeted at  Windows Server 2012 |Targeted at  Windows Server 2008 R2  or  Windows Server 2008 |Targeted at Windows Server 2003|Targeted at Windows 10 Multi-Session on Windows Virtual Desktop (WVD)|
+|-------------------------------|--------------------------------------------|---------------------------------------|------------------------------------|-----------------------------------------------------------------------|------------------|------------------------------------|
+|Windows 10 or Windows Server 2016|Full support|Full support|Full support|After [Software and configuration requirements](#software-and-configuration-requirements) are satisfied, can perform most management tasks, but no role or feature installation or uninstallation|Not supported|Not supported -VM with Win10 E multi session OS is considered a server OS and RSAT only works on client OS|
```

Comment:
I noticed that a regular dash or hyphen is used here to emphasize the main description by adding a further description. I think it would be better to use an em dash here to achieve this goal:
```suggestion
|Windows 10 or Windows Server 2016|Full support|Full support|Full support|After [Software and configuration requirements](#software-and-configuration-requirements) are satisfied, can perform most management tasks, but no role or feature installation or uninstallation|Not supported|Not supported — VM with Win10 E multi session OS is considered a server OS and RSAT only works on client OS|
```
There are (at least) two ways of accessing the em dash while editing text: 1) use the key combination Alt+0151, or 2) open the Character Map app (charmap) and search for the term "em dash", then double-click (or click [Select]) to enter it into the selection box, then click [Copy] to enter it into your clipboard (copy-paste) buffer, for pasting while editing.